enhancements/machine-api/machine-health-checking.md

```diff
@@ -137,6 +150,7 @@ Watch:
 - Watch machines and nodes with an event handler e.g controller runtime `EnqueueRequestsFromMapFunc` which returns machineHealthCheck resources.
 
 Reconcile:
+- Don't start remediation in case the pause annotation is set on the machineHealthCheck resource.
```

Comment:
what is expected from MHC to do with machines that turned from unhealthy to healthy? should it delete the EMR when pause annotation exists?

Reply:
At least upstream reconcilation just doesn't do anything while this annotation exists, and IMHO we should do the same. So the EMR will on;y be removed after unpausing. Will make this more clear, thx!